docs/coderunner.md

```diff
@@ -0,0 +1,37 @@
+## Code Runner Support (currently Linux and MacOS only)
+Tired of typing the same command again and again. Code Runner gives one click execute support for better accessibility, make sure you have the [Code Runner](https://marketplace.visualstudio.com/items?itemName=formulahendry.code-runner) extenstion installed. Then follow the steps given below.
+
+**Note:** You can only run this in easy-masm folder, it will not work dynamically work everywhere. 
+
+**Note:** This method will make the ``` ./run.sh filename ``` method obsolete, see step three to make both steps accessible at the same time.
+
+### 1. Change Extension Settings
+Click on extension settings of the code runner extension.
+
+  ![image](https://github.com/thehamzaihsan/easy-masm-pr/assets/85873694/c60c565e-76e1-402b-bc98-4395380f2ae2)
```

Comment:
Can you move these images to a folder like `docs/images/` so that they're stored on the repo itself as opposed to on your github, and then reference them like that

Reply:
Added the images in docs/images